<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_breakfast</class>
 <widget class="QWidget" name="choose_breakfast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>选择早餐</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <widget class="QListWidget" name="breakfast_menu_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="breakfast_recipe_list">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>双击左侧列表项添加食物
双击右侧列表项移除食物
多份食物请多次双击添加</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yes_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
